src/content/docs/more-integrations/open-source-telemetry-integrations/atlassian/atlassian-integration.mdx

```diff
@@ -106,18 +109,31 @@ After you've selected your New Relic entities, the application starts syncing da
 
 </Steps>
 
-## Enable the incidents tab [#enable-incidents-tab]
+## Enable the incidents feature [#enable-incidents-feature]
+
+If you've project admin permissions, follow these steps to enable the incidents feature:
+
+1. From the sidebar of your project, go to <DoNotTranslate>**Project settings > Features**</DoNotTranslate>.
+
+2. Toggle the <DoNotTranslate>**Incidents**</DoNotTranslate> feature on or off.
+
+3. Once enabled, select <DoNotTranslate>**Incidents**</DoNotTranslate> in your project sidebar to use the feature.
+
+
+See the [incidents feature](https://support.atlassian.com/jira-software-cloud/docs/what-is-the-incidents-feature/) page in the Atlassian Support documentation for more information.
+
+
+## Remove a connection between New Relic and Jira [#remove-connection]
 
-You may need to link to the Atlassian documentation. It walks a Jira site admin through enabling the incidents tab and assigning it to a workspace. There's a chance they are going to enable by default and then, you won't need this section.
+You can stop syncing data from New Relic and remove the synced data at any time. If you've Jira site admin permissions, you can do this from your New Relic configuration page within Jira. You can select your connected organization and click <DoNotTranslate>**Delete**</DoNotTranslate>. Once the process finishes, you will no longer see New Relic incidents and issue information in the incidents feature in Jira.
 
 
-## Remove a connection between New Relic and Atlassian Jira [#remove-connection]
+## Connect to a different organization [#different-conexion]
```

Comment:
@nbaenam What is conexion? Do you mean connection?

Reply:
@ally-sassman, it refers to the connection you have between New Relic and Atlassian Jira. This connection happens when you install the integration, they must be connected to syncing data from New Relic.

Reply:
hahaha, you got me! ;-) I misunderstood your first comment.